### PR TITLE
add linux_ppc64le, linux_aarch64

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
This is ready for review

File: conda-forge.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ github:
 conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```
Comment (hmaarrfk) [inline conda-forge.yml]: can we try cross compiling?

Comment (matthiasdiener) [reply conda-forge.yml]: Would emulation on azure be ok?

Comment (hmaarrfk) [reply conda-forge.yml]: yes since you are helping maintain

**Pull request merged**